### ISSUE TITLE
[Question] What's your workflow with xcake and git?

### ISSUE THREAD

**Issue by chiahsien** [jcampbell05/xcake#141]
Hi all,

Since xcake can generate Xcode project file, do you still commit the project file into git repo?
What's your team's workflow when xcake is involved?
I'd love to get some insights from you all.

**Comment by maximkhatskevich** [comment]
@chiahsien it's up to you, the situation is the same as having CocoaPods dependencies checked-in into your repo or not. If you keep it under version control - you consume extra space in repo (which is nothing when we are talking about Xcode project file), but you less dependent on third-party repos/servers and their availability. You always have a completely working copy and can compile your project at any time just from what you have in your repo. If you don't have your project file in repo, then you depend on having Xcake of the right version installed on each computer where you check-out your repo.

To sum it up, I would recommend to have your project file in your repo, and use Xcake when needed to regenerate it when needed. So if you regenerate it and something does not work - you always can rollback to previous saved project file version.

Hope that helps.

**Comment by jcampbell05** [comment]
For me I live life on the edge and just check in my Cakefile but the above is probably a good thing to do as a fallback especially when working with new xcode versions.

Going to close this for now as I think your question has been answered :)
